assets/shared/data/schedule-controls.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Schedule to run action creator after the given time.
+ *
+ * @param {Function} action Action creator to dispatch.
+ * @param {number} timeout Timeout in milliseconds.
+ */
+export function* schedule( action, timeout ) {
```

Comment:
When I started reading the code, it was a bit confusing for me because I was relating it to the WP cron/Action scheduler. If it makes sense to you too, maybe we could call these functions like `createTimer`, `clearTimer`, and so on.

Reply:
Yeah I was trying to find a good name, but maybe `timeout` is clearer. d3564ad